catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/filter/VisitableXmlElementImpl.java

```diff
@@ -236,15 +241,30 @@ private FunctionElement(JAXBElement element) {
             INVALID_INVOCATION + element.getDeclaredType().getName());
       }
       FunctionType functionType = (FunctionType) element.getValue();
-      this.value =
-          FunctionResolver.resolve(
-              functionType.getName(),
-              functionType
-                  .getExpression()
-                  .stream()
-                  .map(JAXBElement::getValue)
-                  .map(LiteralType.class::cast)
-                  .collect(Collectors.toList()));
+
+      this.value = new HashMap<>();
+      this.value.put("type", "FILTER_FUNCTION");
+      this.value.put("filterFunctionName", functionType.getName());
+      this.value.put(
+          "params",
+          functionType
+              .getExpression()
+              .stream()
+              .map(JAXBElement::getValue)
+              .map(LiteralType.class::cast)
+              .map(
+                  literalType ->
```

Comment:
Extract this lambda to a separate function.

Reply:
done